### PR TITLE
Changes to unittests that verify units for interval and ttl durations. 

### PR DESCRIPTION
**Pull request by sgargan**
##### SUMMARY
Also tests corrected logic for service re-registration

##### ISSUE TYPE
Test Pull Request

##### COMPONENT NAME
plugins/inventory/consul_io.py
test/integration/consul.yml
test/integration/roles/test_consul_service/tasks/main.yml
test/integration/roles/test_consul_session/tasks/main.yml

##### ANSIBLE VERSION
2.3

### CODE REVIEW

File: plugins/inventory/consul_io.py [hunk 1]
```diff
@@ -260,6 +260,7 @@ def load_data_from_service(self, service_name, service, node_data):
     self.add_metadata(node_data, "consul_services", service_name, True)
 
     if self.is_service("ssh", service_name):
+      print 'here'
```
Comment (bcoca) [inline plugins/inventory/consul_io.py:263]: I'm guessing this is a stray debug statement?

Comment (sgargan) [inline plugins/inventory/consul_io.py:263]: Completely...damn how'd that get past the githook

**Comment by amenonsen** [comment]
@sgargan any plans to revise this (to remove the stray print) and rebase to apply to current devel? Note that `consul_io.py` now lives in `contrib/inventory`.

**Comment by samdoran** [comment]
cc @sgargan Any updates on this? We're interested in this if you can correct the issues.

needs_info

**Pull request closed without merging**